llvm/test/Transforms/LoopVersioning/crash-36998.ll

```diff
@@ -0,0 +1,57 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes=loop-versioning -aa-pipeline='' -S < %s | FileCheck %s
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
Does this depend on the X86 triple? if so, it needs to be marked as requiring the X86 backend, otherwise the test will fail when the backend is not built